libraries/AP_VisualOdom/AP_VisualOdom.h

```diff
@@ -49,29 +49,28 @@ class AP_VisualOdom
         Vector3f position_delta;    // position delta (in meters) of most recent update
         uint64_t time_delta_usec;   // time delta (in usec) between previous and most recent update
         float confidence;           // confidence expressed as a value from 0 (no confidence) to 100 (very confident)
-        uint32_t last_update_ms;    // system time (in milliseconds) of last update from sensor
+        uint32_t last_sensor_update_ms;    // system time (in milliseconds) of last update from sensor
+        uint32_t last_sensor_update_ms_i_processed; // timestamp of last sensor update that was processed
```

Comment:
sorry to be picking but can we rename this variable from "last_sensor_update_ms_i_processed" to "last_processed_sensor_update_ms" or similar?

Reply:
Fixed and fixed.